action/action.go

```diff
@@ -119,7 +119,10 @@ func ClassifyActions(actions []SealedEnvelope) ([]*Transfer, []*Execution) {
 }
 
 func calculateIntrinsicGas(baseIntrinsicGas uint64, payloadGas uint64, payloadSize uint64) (uint64, error) {
-	if payloadGas == 0 || (math.MaxUint64-baseIntrinsicGas)/payloadGas < payloadSize {
+	if payloadGas == 0 {
+		return 0, ErrUnderpriced
```

Comment:
0 < minimum gas price, `ErrUnderpriced` is more proper

Reply:
Why is the later condition removed